functionalDB/functionalDB.markdown

```diff
@@ -668,14 +668,14 @@ Our data model is based on accumulation of facts (i.e., datoms) over time. For t
 ### Query Language

```

Comment:
We don't need to repeat "query language" here because it was already identified as such when we said " A commonly used query language influenced by logic programming is _Datalog_ which..."